quotaclimat/data_processing/mediatree/channel_program.py

```diff
@@ -0,0 +1,63 @@
+import modin.pandas as pd
+import logging
+import os
+
+def get_programs():
+    current_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
not perfect, we read for a file instead of using a live API (maybe at next iteration)